packages/gatsby/src/utils/webpack-utils.ts

```diff
@@ -10,6 +10,7 @@ import ReactRefreshWebpackPlugin from "@pmmmwh/react-refresh-webpack-plugin"
 import { getBrowsersList } from "./browserslist"
 import ESLintPlugin from "eslint-webpack-plugin"
 import { cpuCoreCount } from "gatsby-core-utils"
+import { extendDefaultPlugins as svgoExtendDefaultPlugins } from "svgo"
```

Comment:
`gatsby` doesn't depend on `svgo` directly - we would need to either explicitely add it to deps ( https://github.com/gatsbyjs/gatsby/blob/2e1f7e325adfe055f828336c247d71404f461be5/packages/gatsby/package.json#L12-L166 ) or maybe figure out some other way to access that function, I just can't find example of that for `minimizerOptions` :(

Reply:
Fair point. I guess we wouldn't want to introduce that dependency, even though it's implied?
I have a list of the current defaults, so could add that and not use this handy method, it's just more brittle.